File: 윤서/2941.py
# 2941 - 크로아티아 알파벳

import sys
input = sys.stdin.readline

word = input()
alphabets = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z=']
count = 0

for alpha in alphabets:
  if alpha in word:
    count += word.count(alpha)
    
result = len(word) - count*2 + count

print(result)

File: 찬미/구현/2941.swift
import Foundation

// Foundation에 좋은 함수가 있는 것을 발견하였습니다
// replacingOccurrences 함수 사용
// self 중 of에 해당하는 문자열로 with로 변경할 때 사용하는 문자열 치환 메서드

var input = readLine()!
let croatia: [String] = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="]

for i in croatia {
    input = input.replacingOccurrences(of: i, with: "a")
}

var cnt = input.count
print(cnt)
